tools/bpftimetool/main.cpp

```diff
@@ -83,8 +83,11 @@ int bpftime_run_ebpf_program(int id,
 			auto new_prog = bpftime_prog(prog.insns.data(),
 							 prog.insns.size(),
 							 prog.name.c_str());
+			// kernel function can only be used in linux env
+			#if __linux__
 					bpftime::bpftime_helper_group::get_kernel_utils_helper_group()
```

Comment:
`kernel_utils_helper` here means kernel eBPF helpers implemented in userspace by bpftime. Most of them are platform independent and able to run macos. Don't exclude the whole helper group on linux platforms

Reply:
I see, will go through these changes then